scripts/jenkins/centos67/VagrantBuild_CentOS67.sh

```diff
@@ -20,7 +20,7 @@ sed -i s/"-std=c++11"// Configure.pri.in
 PG_VERSION=`ls /etc/init.d | grep postgresql- | sort | tail -1 | egrep -o '[0-9]{1,}\.[0-9]{1,}'`
 
 aclocal && autoconf && autoheader && automake --add-missing && \
-  ./configure --quiet --with-rnd --with-services --with-postgresql=/usr/pgsql-$PG_VERSION/bin/pg_config
+  ./configure --quiet --with-rnd --with-services --with-josm --with-postgresql=/usr/pgsql-$PG_VERSION/bin/pg_config
```

Comment:
This question might be more rhetorical than anything else, do we even need the whole `centos67` directory?

Reply:
No, I don't think we do use it anymore. I was just updating all the files I found. But I *think* I could just delete it.

Reply:
I'm all for that.  We don't need to do that in this PR if we don't want to but we could make a new issue for it.

Reply:
going for it